.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Report a bug in Quarkus
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+(Describe the problem clearly and concisely.)
+
+**Expected behavior**
+(Describe the expected behavior clearly and concisely.)
+
+**Actual behavior**
+(Describe the actual behavior clearly and concisely.)
+
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. 
+2. 
+3. 
```

Comment:
Steps to reproduce are great but a link to a minimal reproducer (e.g. github repo) would be even better ;-)